### ISSUE TITLE
Custom Plots to support Metric vs Metric or Param vs Param or Metric vs Param

### ISSUE THREAD

**Issue by daveraghav** [iterative/vscode-dvc#5216]
Currently the custom plots only support param vs metric. It would be great if the custom plots can support metric vs metric, param vs param, or even metric vs param.

**Comment by shcheklein** [comment]
@daveraghav could you please give a bit more context? how do you use extension, how critical is this feature for you? (It would be great to jump on a call for me to learn more about you scenarios and cases).

**Comment by daveraghav** [comment]
for some of the training algorithms I'm running, that do self-optimisation, for example, automatic feature selection, or instances where I could be running hyperparameter optimisation as part of the training script, the parameters like number of features, or specific algorithm parameters are often part of output of one of the stages. so they're not really the metrics itself but params. so in these cases I would like to plot those outs (metrics as per dvc, but params as per my training process), vs the actual accuracy metrics. In these cases I've had to pre-define the plots, as part of the pipeline. It would be handy to allow plotting as part of custom plots through the extension directly. I hope that makes sense. Please let me know if further context is needed. Happy to jump on a call.

**Comment by daveraghav** [comment]
Was there any update to this issue?

**Comment by shcheklein** [comment]
@daveraghav sorry for the radio-silence here. I'll check with the team if we have capacity to take this. I've DMed you on LI - let's jump on call and discuss the details - it can help a lot.
